docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc

```diff
@@ -1,12 +1,12 @@
 [id='{context}-pro-working-with-templates-on-rhmap']
-= Working with {Feature} Templates in {ProductShortName}
+= Working with {PROJECT_NAME} Templates in {ProductShortName}
```

Comment:
ProductShortName seems to be feedhenry not RHMAP ?

Reply:
This is where things get tricky from a docs POV.

Downstream, {ProductShortName} is RHMAP and is currently being used in Prod, ie: the customer portal.

We use that across multiple books and we cannot change this. The thinking from a downstream POV is that  {PROJECT_NAME} will be WFM. 

So downstream:

// Product content attributes
//
:ProductName: Red Hat Mobile......
:ProductShortName: RHMAP
:ProductRelease: 1
:ProductVersion: 1.0

//***NEW START***
{PROJECT_NAME} = WFM